### PR TITLE
feat: initiator decide the url to use

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

Today switching from one URL to the other is done through project webpack.

**What is the chosen solution to this problem?**

We want the initiator to decide where the resources are.
To support this I have switched the build on top of talend-scripts using the preset react-lib.

So now if you want an initiator for you CDN you can build it using the following env var:

```
TUI_VERSION=5.28.0 BASE_URL=https://jmfrancois.github.io/cdn-assets npm run prepublishOnly
```

A version has been published here: https://talend-inject.surge.sh/TalendWebpackInjector.min.js if you want to test

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: cdn/webpack-inject/package.json [hunk 1]
```diff
@@ -4,15 +4,13 @@
...
   },
   "devDependencies": {
-    "@babel/cli": "^7.11.6",
-    "@talend/scripts-config-babel": "^6.6.1",
-    "babel-plugin-uglify": "^1.0.2",
-    "surge": "^0.21.6",
-    "uglify-js": "^3.11.0"
+    "@talend/scripts-core": "^6.9.0",
+    "@talend/scripts-preset-react-lib": "^6.9.0",
+    "surge": "^0.21.6"
```
Comment (frassinier) [inline cdn/webpack-inject/package.json:13]: is it mandatory?

File: cdn/webpack-inject/src/index.js [hunk 2]
```diff
@@ -26,12 +26,19 @@
...
+	}
+
 	function addScripts(urls) {
 		urls.forEach(function onUrl(url) {
 			const script = document.createElement('script');
-			script.setAttribute('src', url);
+			script.setAttribute('src', prepareURL(url));
 			script.async = false;
-			document.body.appendChild(script);
+			document.head.appendChild(script);
```
Comment (frassinier) [inline cdn/webpack-inject/src/index.js:41]: why?

**Review by frassinier — APPROVED** [review]

File: cdn/webpack-inject/package.json (hunk 1 above)
Comment (jmfrancois) [reply cdn/webpack-inject/package.json]: nop thanks I remove it

File: cdn/webpack-inject/src/index.js (hunk 2 above)
Comment (jmfrancois) [reply cdn/webpack-inject/src/index.js]: oups I think I took an old version

**Pull request merged**